bindata/assets/config/openshift-controller-manager-defaultconfig.yaml

```diff
@@ -1,2 +1,4 @@
 apiVersion: openshiftcontrolplane.config.openshift.io/v1
 kind: OpenShiftControllerManagerConfig
+leaderElection:
+  name: openshift-master-controllers
```

Comment:
for posterity, this corresponds to https://github.com/openshift/openshift-controller-manager/blob/39ebafcc026c27abd1181b38e919076e3ec1b465/pkg/cmd/openshift-controller-manager/controller_manager.go#L85